docs/source/docker-usage.md

```diff
@@ -32,7 +32,7 @@ This runs the `test/` directory from the bats-core repository inside the bats Do
 For test suites that are intended to run in isolation from the project (i.e. the tests do not depend on project files outside of the test directory), you can mount the test directory by itself and execute the tests like so:
 
 ```bash
-$ docker run -it -v "$PWD/test:/test" bats/bats:latest /test
+$ docker run -it -v "$PWD/test:/test" bats/bats:latest /code/test
```

Comment:
I think that is missing /code in the Volume mapping.

Reply:
Quite worrying when I'm unable to copy/paste from my own report...